src/pages/superadmin/tableComponent/index.tsx

```diff
@@ -485,26 +485,28 @@ export const MyTable = ({
                         target="_blank"
                         rel="noopener noreferrer"
                       >
-                        {bounty?.title}
+                        {bounty?.title.length > 50
+                          ? `${bounty?.title.substring(0, 47)}...`
+                          : bounty?.title}
```

Comment:
instead of adding this with js we can instead add the ellipses using CSS 

https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Reply:
not in the scope of bounty so this was just for demo but yeah I can do that there are more changes are required in superadmin page

Reply:
@gouravmpk can you address this?

Reply:
Yes

Reply:
I fixed in [df7a9fc](https://github.com/stakwork/sphinx-tribes-frontend/pull/241/commits/df7a9fc8587b042d34a4549e1ededd17e549f717)